src/update_agent/actor.rs

```diff
@@ -85,8 +85,7 @@ impl Handler<RefreshTick> for UpdateAgent {
         };
 
         let update_machine = state_action.then(move |_r, actor, ctx| {
-            if let Some(interval) = actor.refresh_delay(prev_state) {
-                let pause = Self::add_jitter(interval);
```

Comment:
Just wondering, were we previously applying jittering twice due to this?

Reply:
Sorry I missed this earlier, I believe we were applying jittering twice :(